pkgs/tools/misc/kt/default.nix

```diff
@@ -1,17 +1,21 @@
-{ lib, fetchFromGitHub, buildGoPackage }:
+{ lib, fetchFromGitHub, buildGoModule }:
 
-buildGoPackage rec {
+buildGoModule rec {
```

Comment:
Is there a reason you switched from Package to Module?

Reply:
Per the nixpkgs manual
```
The function buildGoPackage builds legacy Go programs, not supporting Go modules.
```

Reply:
Ah, thanks 👍